CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# dbt_sage_intacct v0.2.0
```

Comment:
Is this a breaking change? I would imagine this could probably be a patch upgrade. What are your thoughts?